actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -39,6 +41,15 @@ def corrections # :nodoc:
     end
   end
 
+  # Raised from `expect!` when an expected parameter is missing or of an
+  # incompatible type.
+  #
+  #     params = ActionController::Parameters.new(a: {})
+  #     params.expect!(:a)
+  #     # => ActionController::ExpectedParameterMissing: param is missing or the value is empty: a
+  class ExpectedParameterMissing < ParameterMissing
+  end
```

Comment:
When defining a class without body, I think it is preferable to use one-line `class Abc; end`

Reply:
I was just keeping with the style on line 85.

Reply:
I see, no strong preferences here, consistency is the main thing